documentation/source/user_section/tutorials.rst

```diff
@@ -1,14 +1,21 @@
 .. _tutorials:
 
-Tutorials and Courses
-#####################
+Tutorials
+#########
 
 .. note::
 
    The following tutorials make use of :ref:`Quality Control (QC) <qc>` reports for inspecting processed image data. Optionally, we also recommend trying out :ref:`fsleyes-instructions` to visualize the data from these tutorials, too.
 
-Tutorial pages
-**************
+Video tutorials
+***************
+
+SCT have a `YouTube channel`_ which contains additional tutorials.
+
+.. _Youtube channel: https://www.youtube.com/playlist?list=PLJ5-Fnq9XpaVgCZfY-GOGJaT0fmZN4vji
```

Comment:
I would recommend putting the YouTube section nearer to the bottom. (A lot of our YouTube videos are probably out of date by now tbh, given the difficulty in maintaining them. And, our text-based tutorials are much more up to date and actively maintained, so I think they should come first.)

Plus, the `.. note::` above (lines 6-8) relates directly to the written tutorials, so they should probably come after one another.

Reply:
I moved back the video tutorial after the written tutorial based on your last information. I tough that it would be more easy to see everything, but if they are not up-to-date we are better to let them on the bottom.